torchvision/models/segmentation/segmentation.py

```diff
@@ -28,7 +28,7 @@ def _segm_model(
     name: str,
     backbone_name: str,
     num_classes: int,
-    aux: bool,
+    aux: Optional[bool],
```

Comment:
it's not strict, but isn't `Optional` constrained to argument with specified default value being `None`?

Reply:
It's because it can get None value from the calling methods... Mypy was complaining about it.